crates/bevy_log/src/lib.rs

```diff
@@ -96,7 +96,7 @@ impl Plugin for LogPlugin {
         {
             let fmt_layer = tracing_subscriber::fmt::Layer::default();
             let subscriber = subscriber.with(fmt_layer);
-            #[cfg(feature = "tracing-chrome")]
+            #[cfg(all(feature = "tracing-chrome", not(feature = "tracing-tracy")))]
```

Comment:
Why can't we enable both at once? They don't seem mutually exclusive, unless the `with` method on `subscriber` overwrites

Obviously we'd have to move the line which sets the subscriber further down, but that's fine

Reply:
The problem is that `subscriber.with(chrome_layer)` returns a different type to `subscriber.with(tracy_layer)` so it's kind of awkward. I managed to find an ok-ish way of handling it. If you have better ideas, please let me know.